content/docs/how-to-guides/connectivity-issues.md

```diff
@@ -7,20 +7,20 @@ title: Connecting with old clients
 
 ```txt
-ERROR: Project name is not specified. Either please upgrade the postgres client library (libpq) for SNI support or pass the project name as a parameter: '&options=project%3D<project-name>'. See more at https://neon.tech/sni
+ERROR: Project ID is not specified. Either please upgrade the postgres client library (libpq) for SNI support or pass the project ID (first part of the domain name) as a parameter: '&options=project%3D<project-id>'. See more at https://neon.tech/sni
 ```
 
 In most cases, this happens if your client library or app does not support the so-called **SNI (Server Name Indication)** mechanism in TLS. See [#Details](Details) for more context and [#Workarounds](Workarounds) for a list of ways to get around this issue.
 
 ## Details
 
-To route incoming connections, we use different domain names for different projects, e.g., to connect to the project named `mute-recipe-239816`, we ask you to connect to _**mute-recipe-239816.cloud.neon.tech**_. However, the Postgres wire protocol does not transfer the server domain name, so we always use only encrypted connections and rely on the so-called **SNI (Server Name Indication)** extension of the `TLS` protocol, which allows a client to indicate what domain name it is attempting to connect to. That is the same mechanism that allows hosting several `https`-enabled websites on a single IP address. `SNI` support was added to the `libpq` (an official Postgres client library) in version 14, released in September 2021. All `libpq`-based clients like Python's `psycopg2` and Ruby's `ruby-pg` should work if `libpq` in the system has a version >= 14.
+To route incoming connections, we use different domain names for different projects, e.g., to connect to the project `mute-recipe-239816`, we ask you to connect to _**mute-recipe-239816.cloud.neon.tech**_. However, the Postgres wire protocol does not transfer the server domain name, so we rely on the so-called **SNI (Server Name Indication)** extension of the `TLS` protocol, which allows a client to indicate what domain name it is attempting to connect to. That is the same mechanism that allows hosting several `https`-enabled websites on a single IP address. `SNI` support was added to the `libpq` (an official Postgres client library) in version 14, released in September 2021. All `libpq`-based clients like Python's `psycopg2` and Ruby's `ruby-pg` should work if `libpq` in the system has a version >= 14.
 
 ## Workarounds
 
-If you face a `Project name is not specified` error and library/application upgrade does not help, we provide several fallback options. When `SNI` domain name info is missing, we need to get it via other means.
+If you face a `Project ID is not specified` error and library/application upgrade does not help, we provide several fallback options. When `SNI` domain name info is missing, we need to get it via other means.
```

Comment:
If you encounter a `Project ID is not specified` error and a library or application upgrade does not help, we provide several fallback options. When `SNI` domain name information is missing, we need to obtain this information through other means.